addins/Cake.Frosting.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.Frosting
```

Comment:
Not an addin

Reply:
I maintain a "blacklist" of Nuget packages that should not be processed by the AddinDiscoverer. I will simply add Cake.Frosting to this list.